packages/just-scripts/src/tasks/upgradeRepoTask.ts

```diff
@@ -19,14 +24,18 @@ export async function upgradeRepoTask() {
   if (rootPath) {
     process.chdir(rootPath);
     const scriptsPath = path.join(rootPath, 'scripts');
-    const scriptsPackageJson = fse.readJsonSync(path.join(scriptsPath, 'package.json'));
+    const scriptsPackageJson = readPackageJson(scriptsPath)!;
     const outdatedStacks = await getOutdatedStacks(rootPath);
 
     outdatedStacks.forEach(stack => {
-      if (scriptsPackageJson.devDependencies[stack.name]) {
-        scriptsPackageJson.devDependencies[stack.name] = options.latest ? `^${stack.latest}` : `^${stack.wanted}`;
-      } else if (scriptsPackageJson.dependencies[stack.name]) {
-        scriptsPackageJson.dependencies[stack.name] = options.latest ? `^${stack.latest}` : `^${stack.wanted}`;
+      if (scriptsPackageJson.devDependencies![stack.name]) {
```

Comment:
hmmm now looking at this might be a bug here... need:

```suggestion
if (scriptsPackageJson.devDependencies && scriptsPackageJson.devDependencies[stack.name]) {
```

Reply:
I added a todo for better error handling in that task all-up, since that bit isn't the only issue.